CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Scrape expiry of Secrets of type `kubernetes.io/tls`. ([#92](https://github.com/giantswarm/cert-exporter/pull/92))
```

Comment:
Can this be described in a way that actually explains to an outsider (a user of the app) what benefit this change adds?

Reply:
sure, i can improve this.

Reply:
great, thank you!